docs/design/audit-v2.md

```diff
@@ -0,0 +1,78 @@
+# Auditing V2: Random Node Selection
+
+## Abstract
+
+This design doc outlines how we will implement a Version 2 of the auditing service.
+With our current auditing service, we are currently auditing per segment of data.
+We need to keep this existing method of auditing, but also add an additional audit selection process that selects random nodes to audit.
+
+## Background
+
+As our network grows, it will take longer for nodes to get vetted.
+This is because every time an upload happens, we send 5% of the uploaded data to unvetted nodes and 95% to vetted nodes.
+When auditing occurs, we currently select a random stripe within a random segment.
+If we're selecting audits at random per byte, every segment has some percentage that went to vetted nodes.
+As more nodes join the network, it will become exponentially less likely that an unvetted node will be audited since most data will be stored on vetted nodes.
+With a satellite with one petabyte of data currently, new nodes will take one month to get vetted, which is way too long.
+
+We want a way to select segments to audit based on statistically randomly selected storage nodes.
+However, the way we select nodes should also be biased toward unvetted nodes.
+Currently, there's not a way to select a stripe based on a storage node.
+
+## Design
+
+We will create an audit observer that uses the metainfo loop, and this observer will use both methods of selection. It will create reservoir samples for segments following the current selection method (randomly across bytes), and also create a small reservoir for each unvetted node.
+
+After selection, the rest of the auditing process will occur the same way (picking a segment, picking a stripe, downloading all erasure shares associated with that stripe).
+The chances of selecting the same stripe are rare, but it's normal/expected behavior if occasionally a stripe is audited more than once.
+
+With both loops, we should have auditing that occurs statistically uniform across both nodes and bytes.
+
+### **Selection via Reservoir Sampling**
```

Comment:
Remove the bold, it's inconsistent with the default styling.

Reply:
it's more visible.

Reply:
Why not make it a higher level heading?

Reply:
because it should be under the design heading.